databases/week3/mongoDB/database.mongodb

```diff
@@ -0,0 +1,118 @@
+// Select the database to use.
+use('mealSharingApp');
+
+
+
+// Insert a few documents into the sales collection.
+db.Meals.insertMany([
+  {
+    'id': 1, 
+    'title': 'Chicken Curry',
+    'description': 'Delicious Indian dish',
+    'location': 'Aarhus',
+    'When': new Date ('2023-04-01T11:30:00Z'),
```

Comment:
Do not forget later that you've made "When" with a capital letter when you'll create queries with it  because MongoDB's default behaviour is case-sensitive when it comes to field names:)